apps/speed.c

```diff
@@ -1610,13 +1427,12 @@ int speed_main(int argc, char **argv)
         {  test1024,  sizeof(test1024),  1024 },
         {  test2048,  sizeof(test2048),  2048 },
         {  test3072,  sizeof(test3072),  3072 },
-        {  test4096,  sizeof(test4096),  4092 },
+        {  test4096,  sizeof(test4096),  4096 },
```

Comment:
:)

Reply:
This was actually a recent regression in master.

Reply:
It is one way to speed it up also :)

Reply:
A comment on the line saying that it was done on purpose might have been good..

Reply:
As the wrong 4092 value was just a regression in master (the 1.1.1 code does not have it), I do not think a comment is needed.